tests/Organization.py

```diff
@@ -429,19 +429,19 @@ def testCreateRepoFromTemplateWithAllArguments(self):
         self.assertEqual(repo.description, description)
         self.assertTrue(repo.private)
 
-    @mock.patch("github.PublicKey.encrypt")
```

Comment:
I think you can remove related test data: tests/ReplayData/Organization.testCreateSecret.txt

Reply:
Removed from repo